pkgs/tools/backup/flockit/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, bash }:
+
+stdenv.mkDerivation rec {
+  name = "flockit-${version}";
+  version = "2012-08-11";
+
+  src = fetchFromGitHub {
+    owner  = "smerritt";
+    repo   = "flockit";
+    rev    = "5c2b2092f8edcc8e3e2eb6ef66c968675dbfa686";
+    sha256 = "0vajck9q2677gpn9a4flkyz7mw69ql1647cjwqh834nrcr2b5164";
+  };
+
+  installPhase = ''
+    mkdir -p $out/lib $out/bin
+    cp ./libflockit.so $out/lib
+
+    (cat <<EOI
+    #!${bash}/bin/bash
```

Comment:
I changed it.

Do note that `#!/bin/sh` is automatically rewritten to `#!${pkgs.bash}/bin/sh` during the post-installation fixups:

```
post-installation fixup
...
/nix/store/fd7sj17pyja2wpm2yh5jzmim8nhkra15-flockit-2012-08-11/bin/flockit: interpreter directive changed from "/bin/sh" to "/nix/store/jjg5zsfjl35b239f6znp0nfa9775qy2a-bash-4.4-p12/bin/sh"
```